Rubberduck.Core/UI/About/AboutControlViewModel.cs

```diff
@@ -30,24 +34,29 @@ public AboutControlViewModel(IVersionCheck version)
 
         public string HostExecutable => string.Format(AboutUI.AboutWindow_HostExecutable,
             Path.GetFileName(Application.ExecutablePath).ToUpper()); // .ToUpper() used to convert ExceL.EXE -> EXCEL.EXE
+            
+        public string AboutCopyright =>
+            string.Format(AboutUI.AboutWindow_Copyright, DateTime.Now.Year);
```

Comment:
Sorry for dancing around this, but @mansellan and I had a [discussion](https://chat.stackexchange.com/transcript/message/49364390#49364390) following this suggestion, and this was the killer argument:

> I suspect that DateTime.Now.Year is out, because if\when RD becomes abandonware it's unjustifiable to claim copyright until the end of time...

I didn't think this through when I suggested using `DateTime.Now.Year` to just make the update automatic: even if RD keeps being maintained until the end of time, it's not reasonable to have a given build suddenly (on Jan 1st) assert copyright for code that isn't in *that* build.

So bottom line, maybe it's better to simply keep updating the resource manually every year to assert copyright of our collective contributions for that year. That said, it doesn't have to be fixed in *this PR*, or even at any point this year -- just something to keep in mind.

Reply:
Just to point out - we already have a build task to update the copyright year. No reason we can't do the same for this case.

Reply:
Just to double-check: are you guys happy with me leaving the copyright year as it currently is in this PR (with the idea of fixing that on the build system in a separate PR later), or should I do any modifications already?

Reply:
By me, it's OK to merge the PR as-is; I have to agree that the copyright year and all that is technically out of scope of the PR and would likely go beyond the UI changes this PR focuses on.

Reply:
Considering how long this PR has been approved and how lacking the response to this very justified question has been, I've gone ahead and merged the PR. Dealing with this may as well be a separate PR